repository-data/webfiles/src/main/resources/site/freemarker/common/hero.ftl

```diff
@@ -99,8 +98,8 @@
                 <div class="nhsd-t-col-s-6 nhsd-!t-no-gutters ${colSize}">
                     <figure class="nhsd-a-image ${imageSize}" aria-hidden="true">
                         <#if isVideoDoc && document.videoUri???has_content>
-                             <div class="nhsd-o-hero__iframe-wrapper">
-                                <iframe class="nhsd-o-hero__iframe" type="text/html" src="${document.videoUri}" frameborder="0" allow="autoplay; encrypted-media; picture-in-picture" allowfullscreen></iframe>
+                             <div style="padding-bottom: 56.25%; position: relative">
+                                <iframe style="width: 100%; height: 100%; position: absolute" type="text/html" src="${document.videoUri}" frameborder="0" allow="autoplay; encrypted-media; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Why have you changed this? If you link to the correct version of the toolkit you will find this implemented https://dbs1dgg3cdf4v.cloudfront.net/storybook/latest/?path=/story/design-system-components-organisms-hero--featured-with-video.